file_paths.md

```diff
@@ -15,39 +15,40 @@ This table specifies the file paths in a Puppet installation and the correspondi
 
 The package will create two services `puppet` and `mcollective`, both
 running as `root` by default. It will not create a `puppet` user or group.
+The files annotated by an '*' indicate that they are created during packaging.
```

Comment:
"indicate that they are created during packaging" => "are created by package installation"  ??